app/models/form.rb

```diff
@@ -11,6 +11,8 @@ class Form
     },
     2023 => {
       submission_deadline: Time.zone.local(2024, 6, 7),
+      new_logs_end_date: Time.zone.local(2024, 7, 14),
+      edit_end_date: Time.zone.local(2024, 7, 14),
```

Comment:
I'll take these out before merge but seems easier to leave them in until everything else is sorted.